File: java/hello.java
public class hello
{
    public static void main(String[] args)
    {
        int i=0;
        while(i<10)
        {
            System.out.println(i);
            i++;
        }
    }
    public static long  fac(int n)
    {
        long s = 1;
        for(int i = 1;i<=n;i++)
        {
            s = s * i;
        }
        return s;
    }
}

File: kotlin/learn/hello.kt
package hello // 加不加都不会报错

class Person(val name : String){
    fun getName(){
        println("name is $name");
    }
}

fun main(){
    Person("yuri is yes!").getName();
    println("可以使用分号!");
    // 加不加分号都可以
}